etl/steps/data/garden/agriculture/2023-04-21/uk_long_term_yields.meta.yml

```diff
@@ -0,0 +1,15 @@
+dataset:
+  title: Long-term yields in the United Kingdom (agriculture, 2023)
```

Comment:
How I have typically structured the name of long-run variables is: `<dataset title> (various sources, <year>`. 

Typically, we would have the source name in parenthesis, but in this case I use `various sources`